fury/molecular.py

```diff
@@ -0,0 +1,729 @@
+    """
+    return molecule.SetBondOrder(bond_index, bond_type)
+
+
+def get_atomic_number_array(molecule):
+    """Returns an array of atomic numbers corresponding to the atoms
+    present in a given molecule.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule whose atomic number array is to be obtained.
+    """
+    return numpy_support.vtk_to_numpy(molecule.GetAtomicNumberArray())
+
+
+def get_bond_types_array(molecule):
+    """Returns an array containing the types of the bond (single/double/
+    triple) corresponding to the bonds present in the molecule.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule whose bond types array is to be obtained.
+    """
+    return numpy_support.vtk_to_numpy(molecule.GetBondOrdersArray())
+
+
+def get_atomic_position_array(molecule):
+    """Returns an array of atomic coordinates corresponding to the atoms
+    present in the molecule.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule whose atomic position array is to be obtained.
+    """
+    return numpy_support.vtk_to_numpy(molecule.GetAtomicPositionArray().
+                                      GetData())
+
+
+def deep_copy(molecule1, molecule2):
+    """
+    Deep copies the atomic information (atoms and bonds) from molecule2 into
+    molecule1.
+
+    Parameters
+    ----------
+    molecule1 : Molecule() object
+        The molecule to which the atomic information is copied.
+    molecule2 : Molecule() object
+        The molecule from which the atomic information is copied.
+    """
+    molecule1.DeepCopyStructure(molecule2)
+
+
+def compute_bonding(molecule):
+    """
+    Uses vtkSimpleBondPerceiver to generate bonding information for a
+    molecule.
+    vtkSimpleBondPerceiver performs a simple check of all interatomic
+    distances and adds a single bond between atoms that are reasonably
+    close. If the interatomic distance is less than the sum of the two
+    atom's covalent radii plus a tolerance, a single bond is added.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule for which bonding information is to be generated.
+
+    Notes
+    -----
+    This algorithm does not consider valences, hybridization, aromaticity,
+    or anything other than atomic separations. It will not produce anything
+    other than single bonds.
+    """
+    bonder = vtk.vtkSimpleBondPerceiver()
+    bonder.SetInputData(molecule)
+    bonder.SetTolerance(0.1)
+    bonder.Update()
+    deep_copy(molecule, bonder.GetOutput())
+
+
+class MoleculeMapper(vtk.vtkOpenGLMoleculeMapper):
+    """Class to create mappers for three types of molecular represenations-
+    1. Ball and Stick Representation.
+    2. Stick Representation.
+    3. Sphere Representation.
+
+    It is used to create mappers which are then used to create actors for the
+    above mentioned representations.
+    """
```

Comment:
What is the goal of this empty class?

Reply:
It's used to create the mappers which are then used to create actors. Please see [here](https://github.com/fury-gl/fury/pull/452/files#diff-8cedb8d1d2aca3f6f359201c5623b21f1f9b55329c4e7cae44d396e9a62bdec5R632) to see how it's used.

Reply:
It is strange to have an empty class. I do not think you need it. 

I should use directly vtkOpenglMoleculeMapper in your different function

Reply:
Okay! I'll use vtkOpenGLMoleculeMapper directly.